nservicebus/hosting/ibus-instance.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Managing IBus instance
+summary: Fluent configuration API to get transnational one-way messaging
+tags:
+- Hosting
+- IOC
+- Dependency Injection
+- Service Locator
+---
+
+
+When you have created a `IBus` instances then you must register it somewhere so that your application can use it to send messages.
+
+There are various ways in managing and storing the `IBus` instance.
+
+* Global static
```

Comment:
The Global static way of doing this should be moved to the end. An example is needed of when you would recommend its usage.